examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<p>and 1s everywhere else (avoid describing function values as the grid)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>